.gitignore

```diff
@@ -24,3 +24,4 @@ EXAMPLES.md
 Function_Changes.md
 .vscode
 _build
+pyobis/_version.py
```

Comment:
We don't want to track this file. It should be shipped only on tagged releases but no need to keep it in the repo.